### PR TITLE
George -  settings tests

### PR DESCRIPTION
**Pull request by georgezsiga**

### CODE REVIEW

File: app/src/test/java/com/greenfox/tribesoflagopusandroid/SettingsFragmentTest.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    @Test
+    public void turnNotificationSwitchOn() {
+        SettingsFragment settingsFragment = new SettingsFragment();
+        startFragment(settingsFragment);
+        assertNotNull(settingsFragment);
+        settingsFragment.notification.setChecked(false);
+        settingsFragment.notification.performClick();
+        String message = "You will receive game notifications";
+        assertEquals(message, settingsFragment.notification_status.getText());
+        assertEquals("true", settingsFragment.preferences.getString(NOTIFICATION, ""));
```
Comment (adamgyulavari) [inline app/src/test/java/com/greenfox/tribesoflagopusandroid/SettingsFragmentTest.java]: @georgezsiga somehow I missed this, but pls use boolean in the preferences

File: app/src/test/java/com/greenfox/tribesoflagopusandroid/SettingsFragmentTest.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+/**
+ * Created by georgezsiga on 6/23/17.
+ */
+@Config(constants = BuildConfig.class)
+@RunWith(RobolectricTestRunner.class)
+public class SettingsFragmentTest extends android.app.Activity {
+
+    @Test
+    public void turnNotificationSwitchOn() {
+        SettingsFragment settingsFragment = new SettingsFragment();
```
Comment (adamgyulavari) [inline app/src/test/java/com/greenfox/tribesoflagopusandroid/SettingsFragmentTest.java:25]: @georgezsiga also move the `SettingsFragment` into the `fragments` package

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by georgezsiga** [comment]
done

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java [hunk 3]
```diff
@@ -23,8 +23,8 @@
 public class MainActivity extends AppCompatActivity {
 
     public static final String USERNAME = "Username";
-    public static final String NOTIFICATION = "Notification";
-    public static final String BACKGROUND_SYNC = "BackgroundSync";
+    public static final Boolean NOTIFICATION = false;
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/MainActivity.java]: @georgezsiga the preference key should be string, so this was fine like that

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/SettingsFragment.java [hunk 4]
```diff
@@ -45,8 +48,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         background_sync_status = (TextView) rootView.findViewById(R.id.background_sync_status);
         background_sync = (Switch) rootView.findViewById(R.id.background_sync);
 
-        notification.setChecked(Boolean.parseBoolean(preferences.getString(NOTIFICATION, "")));
-        background_sync.setChecked(Boolean.parseBoolean(preferences.getString(BACKGROUND_SYNC, "")));
+        notification.setChecked(preferences.getBoolean(String.valueOf(NOTIFICATION), false));
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/fragments/SettingsFragment.java]: @georgezsiga here the preference value should have been a boolean, so this should look like:
`preferences.getBoolean(NOTIFICATION, false)`

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by georgezsiga** [comment]
done

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**